src/frontend/src/btc/convert/ConvertToCkBTC.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	import { BTC_MAINNET_TOKEN } from '$env/tokens.btc.env';
+	import IconCkConvert from '$lib/components/icons/IconCkConvert.svelte';
+	import ButtonHero from '$lib/components/ui/ButtonHero.svelte';
+	import { isBusy } from '$lib/derived/busy.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+</script>
+
+<ButtonHero disabled={$isBusy} ariaLabel={$i18n.convert.text.convert_to_ckbtc}>
+	<IconCkConvert size="28" slot="icon" />
+	<span>{BTC_MAINNET_TOKEN.twinTokenSymbol}</span>
```

Comment:
we have `ckTestBTC` as token, no? wouldn't it better to use the `$pageToken` store in this case? (not inside the modal I mean, just for the button)

or just for mainnet indeed?

Reply:
For the time being, my assumption is that I implement it only for mainnet. After all the UI and logic for mainnet is finished, this will be an easy thing to make it available for testnet too. WDYT?

Reply:
understood, thank you for the explanation!